Readme.md

```diff
@@ -0,0 +1,48 @@
+# Okta API Access Management Sample Resource Server (ASP.NET Core)
+
+The code sample available in this repository demonstrate the use of [Okta API Access Management](http://developer.okta.com/docs/api/resources/oauth2.html) for authorizing access to a backend API using Okta's Authorization Servers.
+
+This ASP.NET Core OAuth 2.0 Resource Server will only accept calls to its TodoController if 
+1. the incoming requests contains the proper _Bearer_ OAuth token (i.e. properly validated as originating from the expected Okta organization, OpenID Connect client application and audience)
```

Comment:
This doesn't format correctly in github - you probably need an extra line before the bullet list. I recommend using  [grip](https://github.com/joeyespo/grip) locally to see the exact rendering.